lsp-methods.el

```diff
@@ -940,7 +940,15 @@ https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#co
       :annotation-function #'lsp--annotate
       :display-sort-function #'lsp--sort-completions)))
 
-;;; TODO: implement completionItem/resolve
+(defun lsp--resolve-completion (item)
+  (lsp--cur-workspace-check)
+  (lsp--send-changes lsp--cur-workspace)
+  (cl-assert item nil "Completion item must not be nil")
+  (if (gethash "resolveProvider" (lsp--capability "completionProvider"))
+      (lsp--send-request (lsp--make-request
+			  "completionItem/resolve"
+			  item))
+    item)
```

Comment:
I think you need another paren here to close out the `defun` (CI failed because of this).

Reply:
Ah... sorry 'bout that. Sometimes you just forget those parenthesis :P

Reply:
I understand! I highly recommend a mode like `paredit` for editing Lisps.

Reply:
I usually use smartparens, but it just didn't happen for whatever reason that time.